### PR TITLE
CU-8669f83y4 | CU-8669f83yw | CU-8669f83zb | Implemented the home screen music carousels

### PR DESCRIPTION
**Pull request by Jermainelr**
Screenshot1
Screenshot2

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-8669f83y4 Build  “Explore New Music” Carousel](https://app.clickup.com/t/8669f83y4)

**Comment by newmskywalker** [comment]
Task linked: [CU-8669f83yw Build  “Recently Played” Carousel](https://app.clickup.com/t/8669f83yw)

**Comment by newmskywalker** [comment]
Task linked: [CU-8669f83zb Build  “Just Released” Carousel](https://app.clickup.com/t/8669f83zb)

File: android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    musicModels: List<MusicModel>,
+    onViewDetails: (MusicModel) -> Unit,
+    onViewAll: (() -> Unit)? = null
+) {
+    Column(
+        modifier = Modifier
+            .padding(top = 28.dp)
+            .fillMaxWidth()
+    ) {
+        Row(modifier = Modifier.padding(start = 12.dp, end = 12.dp)) {
```
Comment (cristhianescobar) [inline android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt]: For places where you need to add padding to the start and end, you could use something like horizontal / vertical
`Modifier.padding(horizontal = 12.dp)`

File: android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+@Composable
+private fun MusicCarouselItem(
+    musicModel: MusicModel,
+    onClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .padding(start = 6.dp, end = 6.dp)
```
Comment (cristhianescobar) [inline android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt]: Same

File: android/app-newm/src/main/java/io/newm/screens/home/ThisWeekCarousel.kt [hunk 3]
```diff
@@ -30,7 +30,7 @@ fun ThisWeekCarousel(
 ) {
     Column {
         Row(
-            modifier = Modifier.padding(start = 12.dp, end = 12.dp),
+            modifier = Modifier.padding(start = 12.dp, end = 12.dp)
```
Comment (cristhianescobar) [inline android/app-newm/src/main/java/io/newm/screens/home/ThisWeekCarousel.kt]: Same

**Review by cristhianescobar — APPROVED** [review]
LGTM, left some minor comments around padding.
The UI looks fantastic @Jermainelr

File: android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt (hunk 1 above)
Comment (Jermainelr) [reply android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt]: Done!

**Pull request merged**